src/os/exec_other.go

```diff
@@ -0,0 +1,21 @@
+//go:build !aix && !android && !freebsd && !linux && !netbsd && !openbsd && !plan9 && !solaris
+// +build !aix,!android,!freebsd,!linux,!netbsd,!openbsd,!plan9,!solaris
+
+package os
```

Comment:
I think you need to define `Interrupt` and `Kill` here to get the Windows and MacOS CI to pass.